terraform/destroy.sh

```diff
@@ -28,6 +28,14 @@ acct=$(gcloud info --format="value(config.account)")
 SCRIPT_DIR=$(realpath $(dirname "$0"))
 cd $SCRIPT_DIR
 
+# For the purposes of telemetry
+# session is defined as the current time the user runs destroy.sh file
```

Comment:
don't we record time too?

Reply:
We do; I meant current instance. I will clarify that in my documentation.